portfolio/src/main/webapp/aboutMe.html

```diff
@@ -2,15 +2,24 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
     <title>About Me</title>
   </head>
   <body>
-    <h1>About Me</h1>
-    <p>I am Isabelle Dittamo!</p>
-    <p>I am an intern at Google as a part of the STEP Internship for the Summer of 2020!</p>
-    <img src="images/googleGLogo.jpg" width="200"/>
-    <p>I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science with concentrations in Information Internetworks and Intelligence.</p>
-    <img src="images/gtLogoBuzz.png" width="200"/>
-    <p>At school, I am a Teaching Assistant in the College of Computing! I have previously TA'ed for the Introduction to Computing course.</p>
+    <div id="content">
+        <h1>About Me</h1>
+        <p>I am Isabelle Dittamo!</p>
+        <p>I am an intern at Google as a part of the STEP Internship for the Summer of 2020!</p>
+        <img src="images/googleGLogo.jpg" width="200"/>
+        <p>I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science with concentrations in Information Internetworks and Intelligence.</p>
```

Comment:
Generally you should character limit each line. Say 100 characters?

This helps readability for the reviewer and subsequent readers of your code.

Unfortunately, I couldn't find a way to enable a suggestion line in Cloud Shell (this is something most IDEs offer).

Reply:
Got it! Making those changes right now.